### DISCUSSION TITLE
bash: /usr/bin/sudo: Permission denied

### DISCUSSION THREAD

**Discussion by hublex** [netblue30/firejail#5972] [Q&A] [answered]
my plan was, making my life more easier. I do use a LTE-Router atm with limited data volume only. To not let any app entering the internet on my Ubuntu System and using my limited resources I was hoping to use use your app choose only a few apps which are allowed to enter the internet, like internet browser...
Now I am blocked using my system as usual, like opening my files on my cloud. 
I neither able to open firejail for changing the settings, nor able to de install it.

failing to unistall:

```
sudo firecfg --clean
bash: /usr/bin/sudo: Permission denied
```

type into the terminal firejail and I get the following answer:

```
Warning: an existing sandbox was detected. /bin/bash will run without any additional sandboxing features
Error: --shell=none configured, but no program specified
```

Please help

**Comment by kmk3** [comment]
> failing to unistall:
> 
> ```
> sudo firecfg --clean
> bash: /usr/bin/sudo: Permission denied
> ```
> 
> type into the terminal firejail and I get the following answer:
> 
> ```
> Warning: an existing sandbox was detected. /bin/bash will run without any additional sandboxing features
> Error: --shell=none configured, but no program specified
> ```

What is the output of the following commands?

```sh
firejail --version
which -a firecfg sudo
ls -l /usr/bin/sudo
```

**Comment by rusty-snake** [comment]
Wirst case recovery: Boot with `init=/bin/bash` (or maybe systemds rescue/emergency targets), remove all symlink in /use/local/bin pointing to firejail and chsh the shell of your user. If you use a SELinix system relabel on next boot.

Before doing anything of the above, read (arch)wiki and understand what the commands do.

**Comment by aritus007** [comment, marked as answer]
`bash: /usr/bin/sudo: Permission denied`

I did uninstall and reinstalled Firejail to have this issue resolved.  

`$ sudo firecfg --clean`
`$ sudo apt remove firejail`
`$ sudo apt purge firejail`

The reinstall.
